Week3/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Great idea to add the package.json, this makes installation of your code dependencies super straightforward.